pkg/actions/bridge/urfavecli.go

```diff
@@ -22,7 +22,19 @@ func actionUrfavecliV3(command ...string) carapace.Action {
 				if len(lines) <= 1 {
 					return carapace.ActionFiles()
 				}
-				return carapace.ActionValues(lines[:len(lines)-1]...).NoSpace([]rune("/=@:.,")...)
+				vals := make([]string, 0)
+				for _, line := range lines {
+					if line != "" {
+						splitted := strings.SplitN(line, ":", 2)
+						name := splitted[0]
+						description := ""
+						if len(splitted) > 1 {
+							description = splitted[1]
+						}
+						vals = append(vals, name, description)
```

Comment:
[`strings.Cut`](https://pkg.go.dev/strings#Cut) makes this a little easier
```suggestion
						name, description, _ := strings.Cut(line, ":")
						vals = append(vals, name, description)
```

Reply:
Nice, forgot this existed. Updated.